lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -80,6 +82,11 @@ def self.available_options
                                        description: 'Skips confirmation',
                                        is_string: false, # true: verifies the input is a string, false: every kind of value
                                        default_value: false), # the default value if the user didn't provide one
+          FastlaneCore::ConfigItem.new(key: :app,
+                                       env_name: 'APP',
```

Comment:
That's quite a short name for an env var, given that env vars are all sharing a global space. The convention is usually to use some prefix like `FASTLANE_APP_NAME` or similar

Reply:
Also, worth noting that introducing this new key will be a breaking change in the toolkit.
It will default to `wordpress` so this means it won't break WordPress, but it will break woocommerce-android.

Don't we already have an ENV var in our current fastfiles (`PROJECT_NAME` or something like that iirc?) containing this value? If so, using this existing env var name as `env_name` here would make this portable and non-breaking.